locations/spiders/red_e.py

```
@@ -0,0 +1,100 @@
+    extras = {f"socket:{socket}": str(count) for socket, count in counts.items()}
+    extras.update({f"socket:{socket}:output": f"{output:g} kW" for socket, output in outputs.items()})
+    extras["capacity"] = str(sum(counts.values()))
+    return extras
+
+
+class RedESpider(Spider):
+    name = "red_e"
+    item_attributes = {
+        "brand": "Red E",
+        "operator": "Red E",
+    }
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)